gpio_linux.go

```diff
@@ -122,6 +123,9 @@ func OpenPin(n int, mode Mode) (Pin, error) {
 		valueFile: value,
 		initial:   true,
 	}
+	if exposed {
```

Comment:
exposed is set to true if GPIO port is already opened :

`func expose(pin int) (string, bool, error) {
	exposed := false
	pinBase := filepath.Join(gpiobase, fmt.Sprintf("gpio%d", pin))
	var err error
	if _, statErr := os.Stat(pinBase); os.IsNotExist(statErr) {
		err = writeFile(exportPath, "%d", pin)
	} else {
		exposed = true
	}
	return pinBase, exposed, err
}`